deliverables/README.md

```diff
@@ -31,38 +40,96 @@ What is the role of Javascript in a web page?
+
 What are tags?  
+Tags mark the beginning and end of an element. 
+
 What are attributes?  
+Attributes are extra pieces of information in an element, included inside the opening tag. 
+
 What are forms?  
+Forms are an interaction point between the user and the web page or application.  
+
 What is a div?  
+A division, or section, in an HTML document. 
 
 What are selectors?  
+Selectors are things that can be edited, like a word, paragraph, link, etc. Basically anything in HTML can be a selector.
+
 What are properties?  
+Properties are the aspects being edited: font, color, font-family, font-size, height, width, position, margin, etc. 
+
 What are values?  
+Values are the specifics given to the properties. Color: red, for example. 
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+CSS styles are inherited from the "parent". If you definte another style later, the later one is chosen instead. 
+
 What are two CSS attributes you can change to push an element around on the page?  
+Padding and margins. 
+
 What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+1. Link the external stylesheet to HTML 
+2. Internal stylesheet written into HTML 
+3. Inline styles 
 
 ---
 ### Section 3: Git and Github  
 What is Git?  
+A git is a distributed revision control system.  
+
 What is SCM?  
+Source code management system. 
+
 What is a VCS?  
+Version control systems.  
+
 Why is Git useful for a developer?  
+It's able to handle all types of projects making it a versatile tool. 
+
 Why is Git useful for a team of developers?  
+Git is useful for a team of developers because it allows multiple developers to work together on a feature before pushing the work to origin.  
+
 How do you create a new Git repository for a project locally?  
+In Terminal, change the current working directory to your local project, initialize it as a Git repository, add and commit the files to local repo. 
+
 How do you create it on Github?  
+Click "new repository", name it, desribe it, make it public or private, initialize with a README file, create repo.  
+
 How do you commit changes?  
+In Github you click "commit" locally, you have to use the terminal. 
+
 What is the difference between staging and committing changes?  
+Staging is the set up/preview for committing, to make sure you're committing the right projects. Committing is the final step.
+
 What is the difference between committing your changes and pushing them to Github?  
+You first have to commit your project locally before pushing to Github. 
+
 What is the command to check the status of your current repo in git?  
+git status
+
 What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
+git log, git diff, git merge
```

Comment:
git log and git diff will show your commit history and the changes between your commits - git merge will actually merge your branch.